extensions/amp-auto-ads/0.1/ad-network-config.js

```diff
@@ -97,6 +97,7 @@ class AdSenseNetworkConfig {
     return buildUrl('//pagead2.googlesyndication.com/getconfig/ama', {
       'client': this.autoAmpAdsElement_.getAttribute('data-ad-client'),
       'plah': canonicalHostname,
+      'url': docInfo.canonicalUrl,
       'ama_t': 'amp',
```

Comment:
Good point. We already limit the length of the URL, but have now moved URL to be the last param so that if truncation happens we don't lose the more important params.